libsolidity/formal/SolverInterface.h

```diff
@@ -65,6 +65,27 @@ class Expression
 	Expression& operator=(Expression const&) = default;
 	Expression& operator=(Expression&&) = default;
 
+	/// Checks whether this operator has a valid arity based on its name.
```

Comment:
The comments suggests that the name of the function is not the best ;)

`hasCorrectArity`?